arithmetization/src/main/java/net/consensys/linea/plugins/continoustracing/ContinuousTracingBlockAddedListener.java

```diff
@@ -59,7 +60,9 @@ public void onBlockAdded(final AddedBlockContext addedBlockContext) {
           try {
             final CorsetValidator.Result traceResult =
                 continuousTracer.verifyTraceOfBlock(
-                    blockHeader.getNumber(), blockHash, new ZkTracer());
+                    blockHeader.getNumber(),
+                    blockHash,
+                    new ZkTracer(Bytes.fromHexString("c0ffee").toUnsignedBigInteger()));
```

Comment:
I don't understand how this chain id makes sense, but this is exactly what was happening before.  I think perhaps we should just use `LINEA_MAINNET` here?

Reply:
IIRC this value `c0ffee` was chosen to be recognizable and to fix a NPE in certain (?) unit tests.